hack/lib/swagger.sh

```diff
@@ -116,7 +116,9 @@ kube::swagger::gen_api_ref_docs() {
       --rm -v "${tmp_in_host}":/output:z \
       -v "${swagger_spec_path}":/swagger-source:z \
       -v "${register_file}":/register.go:z \
-      --net=host -e "https_proxy=${KUBERNETES_HTTPS_PROXY:-}" \
```

Comment:
this is a slight change in behavior (using the normal proxy vars instead of `KUBERNETES_HTTPS_PROXY`).

I'm OK with this, but we might want to remove the other instances of `KUBERNETES_HTTPS_PROXY` too (including in `build/README.md`).

Reply:
Yes, if there is no objections, I'd update patch to respect everywhere standard variables (HTTP_PROXY/http_proxy). there is no sense for user to specify multiple times same information if he is behind restrictive firewall.